src/Components/OATGraphViewer/Internal/OATGraphCustomEdge.tsx

```diff
@@ -89,44 +216,19 @@ const OATGraphCustomEdge: React.FC<IOATGraphCustomEdgeProps> = ({
             const sourceBetaAngle = Math.asin(sourceBase / sourceHypotenuse);
             const alphaAngle = Math.asin(triangleHeight / triangleHypotenuse);
             const betaAngle = 1.5708 - alphaAngle;
```

Comment:
What's `1.5708`? Can we please define it as a well named constant?

Reply:
Its the Radian Value for 90 degree angles. I'll name it as a constant